app/backend/src/tests/test_api.py

```diff
@@ -34,12 +34,13 @@ def test_ping(db):
     assert res.user.username == user.username
     assert res.user.name == user.name
     assert res.user.city == user.city
+    assert res.user.hometown == user.hometown
```

Comment:
test is failing here because `res.user.hometown` is an empty string, while `user.hometown` is `None`. Maybe this also applies to `pronouns` below?

Reply:
I think a fix for this is for all the fields which are nullable in the database but are not `*Value`s, add some default data in  `generate_user`.

Reply:
I think the problem is in api.proto as far which object types I use. I need to use nullable object types to correspond with the model. I'm uploading a change, let me know if it solves this issue.

Also, what do you mean by "*Value"?